### DISCUSSION TITLE
Time slots for Commonalities Call

### DISCUSSION THREAD

**Discussion by rartych** [camaraproject/Commonalities#72] [Polls]
Dear Participants,
During the last call we have discovered that current time slot 4-5pm CET on Thursday conflicts with other High priority CAMARA meetings.  I think the starting time 4pm CET is suitable for participants from Europe and North America.
Therefore I propose to select it on other day than Thursday: 

- on Monday at 4pm CET starting from 16th of October;
- on Tuesday at 4pm CET starting from 10th of October - needs to be aligned with EdgeCloud subproject calls;
- on Wednesday at 4pm CET starting from 11th of October - needs to be aligned with Identity&Consent subproject calls and requires shifting of Carrier Billing calls ;

We can also stay with the current slot and shift the call to 3pm CET or other week when conflict occurs.

Please provide your votes and indicate your preferences in comments by **Monday 9th of Oct EoB**

**Comment by bigludo7** [comment]
Thanks @rartych 
Voted for Tuesday but I can make it also the Monday.
